hasher-matcher-actioner/src/OpenMediaMatch/blueprints/hashing.py

```diff
@@ -72,11 +72,20 @@ def hash_media_post():
     """
     Calculate the hash for the provided file.
     """
-    return hash_media_post_impl()
 
+    return hash_file()
 
-def hash_media_post_impl() -> dict[str, str]:
-    """ """
+
+def hash_file() -> dict[str, str]:
```

Comment:
blocking: The old name actually was meaningful in that it was referring to the fact that it is one variation of the hash_file implementation - one where the media content is in the form data. There's another version which downloads from a URL. This new name is probably misleading because someone might confuse it with how `tx hash <file>` works.

Maybe something like `hash_media_from_form_data` might be more accurate? (I believe form data can only be included in a POST)